File: Tokopedia/Classses/Object/Shops/Notes/NoteDetail/NoteDetails.m
//
//  NoteDetails.m
//  Tokopedia
//
//  Created by IT Tkpd on 10/14/14.
//  Copyright (c) 2014 TOKOPEDIA. All rights reserved.
//

#import "NoteDetails.h"

@implementation NoteDetails

- (NSString*)notes_title {
    return [_notes_title kv_decodeHTMLCharacterEntities];
}

- (NSString*)notes_content {
    return [_notes_content kv_decodeHTMLCharacterEntities];
}

@end

File: ios/Tokopedia/Features/Object/Shops/Notes/NoteDetail/NoteDetails.swift
//
//  NoteDetails.swift
//  Tokopedia
//
//  Created by Tedo Pranowo on 7/5/16.
//  Copyright © 2016 TOKOPEDIA. All rights reserved.
//

import Foundation
import RestKit

class NoteDetails : NSObject, TKPObjectMapping
{
    var notes_position : NSString = ""
    var notes_status : NSString = ""
    var notes_create_time : NSString = ""
    var notes_id : NSString = ""
    var notes_title : NSString = ""
    var notes_active : NSString = ""
    var notes_update_time : NSString = ""
    var notes_content : NSString = ""
    
    class func attributeMappingDictionary() -> [AnyHashable: Any]! {
        return ["notes_position" : "notes_position",
                "notes_status" : "notes_status",
                "notes_create_time" : "notes_create_time",
                "notes_id" : "notes_id",
                "notes_title" : "notes_title",
                "notes_active" : "notes_active",
                "notes_update_time" : "notes_update_time",
                "notes_content" : "notes_content"]
    }
    
    class func mapping() -> RKObjectMapping! {
        
        let mapping : RKObjectMapping! = RKObjectMapping(for: self)
        
        mapping.addAttributeMappings(from:attributeMappingDictionary())
        
        return mapping
    }
    
}
